src/app/components/pages/my-lists/my-lists.component.html

```diff
@@ -28,11 +28,14 @@ <h4 class="themeFontColorMuted mb-0 d-inline pb-1">Create List</h4>
 <ng-template #mainView>
     <div class="row mt-2 mt-sm-4 mb-4" id="listRow">
         <div class="col-12">
-            <div class="row  d-flex justify-content-center">
-                <div class="col-12 col-sm-4 mb-sm-4" *ngFor="let list of allLists">
-                    <app-list-overview-item [editable]="editable" [list]="list" [deleteCallback]="listDeleted"></app-list-overview-item>
+            <div class="row d-flex justify-content-center">
+                <div class="card-columns w-100">
+                    <div class="card mb-4" *ngFor="let list of allLists">
```

Comment:
Should the card class be in the app-list-overview-item component? Would we ever not want that class on the component?